src/controllers/loginController.ts

```diff
@@ -1,26 +1,52 @@
 import { Request, Response } from 'express';
-import jwt from 'jsonwebtoken';
-import { sendResponse } from '../utils';
-import config from '../config/config';
+import { sendResponse, signToken } from '../utils';
+import { sendEmail } from '../utils/email';
 import { checkLoginCredentials } from '../middleware/loginMiddleware';
 
 /// ///////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 const loginController = async (req: Request, res: Response) => {
   try {
     const { email, password } = req.body;
-    // passing password and email  to middleware function
     const user = await checkLoginCredentials(email, password);
     if (!user) {
-      return sendResponse<null>(res, 400, null, 'User not found');
+      return sendResponse<null>(
+        res,
+        400,
+        null,
+        'User not found or Invalid Credentials'
+      );
     }
+    if (!user.verifiedAccount) {
+      const verificationToken = signToken({ email: user.email }, '15m');
```

Comment:
please use verify

Reply:
And about verify function, I have a file in utils containing function , one used to sign a token, and another used to verify a token. So that we don't need to be calling jwt everytime in the logins or elsewhere.

So yeah, I use verify

Reply:
no that's what I meant I meant the column use verify it's was added by @Nadinefiona